### PR TITLE
Redesign for main-page

### PR DESCRIPTION
**Pull request by Fusionqq**
screen shot 2017-07-28 at 16 50 21

### CODE REVIEW

**Comment by tyndria** [comment]
Please, move all images to the folder src/images

File: src/main-page/main-page.scss [hunk 1]
```diff
@@ -6,6 +6,11 @@
   }
 }
 
+//background image
+body {
+  background: url('../swirl_pattern.png');
```
Comment (Drapegnik) [inline src/main-page/main-page.scss]: @tyndria how we can import images in `scss` without hardcore?

Comment (tyndria) [inline src/main-page/main-page.scss]: Yes, yes, I understood, that there is a problem. I don't know for this moment, frankly

**Comment by tyndria** [comment]
There is not very nice solution with path to image from bundle.css (https://github.com/babajka/babajka-markup/commit/584fa28ad032ae67ef70729df9232898d0d3ed00), but I don't know another one for this moment

**Pull request merged**